src/main/java/org/symphonyoss/symphony/messageml/elements/MessageML.java

```diff
@@ -112,6 +116,7 @@ public void validate() throws InvalidInputException {
     if (format == FormatEnum.MESSAGEML) {
 
       assertNoAttributes();
+      assertNoFormsWithSameId();
```

Comment:
I don't think this should be put in here. This class is to represent the "messageML" tag, not language. 

Also, this validation shouldn't be done only when the format is MessageML, that's not why the format is used on Elements. It's always "MessageML" when being parsed using the MessageMLParser, but the format of the tag might be presentationML directly or messageML. Both would need the validation for formId.

Either way, This shouldn't be done on the MessageML tag, specially as the validation done here only looks at the direct child of the messageML tag, which doesn't consider other elements with forms inside of it.